mojap_metadata/converters/glue_converter/__init__.py

```diff
@@ -12,6 +15,49 @@
 import importlib.resources as pkg_resources
 from dataclasses import dataclass
 from mojap_metadata.converters.glue_converter import specs
+from mojap_metadata.logging_functions import logging_setup
+
+
+logger = logging_setup()
+
+
+# decorator for replacing arguemnts with object properties
+def _inject_properties_factory(obj_name: str, props: List[str]):
```

Comment:
I would drop this. Think it adds too much complexity.

Reply:
Will drop in the glue converter update